src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -65,6 +77,18 @@ public function iAmLoggedInAs($email)
         $this->securityService->logIn($email, 'main', $this->minkSession);
     }
 
+    /**
+     * @Given I am logged in as administrator
+     */
+    public function iAmLoggedInAsAdministrator()
+    {
+        $this->prepareSessionIfNeeded();
+
+        $this->userContext->thereIsAdministratorAccount();
```

Comment:
We shouldn't use contexts in contexts. This should be done through factory (see [my comment](https://github.com/Sylius/Sylius/pull/4009/files#r53003791))